File: src/main/java/frc/robot/commands/Intake_Commands/IntakeArmUp.java
package frc.robot.commands.Intake_Commands;

import com.revrobotics.CANSparkBase;
import edu.wpi.first.wpilibj2.command.Command;
import frc.robot.Constants;
import frc.robot.subsystems.Intake;

public class IntakeArmUp extends Command {
    private Intake intake;

    public IntakeArmUp(Intake intake) {
        this.intake = intake;
        addRequirements(intake);
    }

    @Override
    public void initialize() {
        // intake.getRaiseLowerSpinner().getPIDController().setReference(Constants.INTAKE_ARM_UP, CANSparkBase.ControlType.kSmartMotion);
        intake.getRaiseLowerSpinner().set(Constants.INTAKE_ARM_UP);
    }

    @Override
    public void execute() {
    }

    @Override
    public void end(boolean interrupted) {
    }

    @Override
    public boolean isFinished() {
        return false;
    }
}

File: src/main/java/frc/robot/commands/Intake_Commands/IntakeArmUp.kt
package frc.robot.commands.Intake_Commands

import com.revrobotics.CANSparkBase
import edu.wpi.first.wpilibj2.command.Command
import frc.robot.Constants
import frc.robot.subsystems.Intake

class IntakeArmUp(private val intake: Intake) : Command() {

    init { addRequirements(intake) }

    override fun initialize() {
        intake.raiseLowerSpinner.pidController.setReference(Constants.INTAKE_ARM_UP, CANSparkBase.ControlType.kSmartMotion)
    }

    override fun execute() { }

    override fun end(interrupted: Boolean) {}

    override fun isFinished(): Boolean { return false }

}
